packages/react-server-cli/package.json

```diff
@@ -48,6 +48,9 @@
   },
   "devDependencies": {
     "babel-preset-react-server": "^0.4.4",
+    "gulp": "^3.9.1",
+    "gulp-babel": "^6.1.2",
+    "gulp-eslint": "^3.0.1",
```

Comment:
Yeah, this is great.

We've gone back and forth here.  It's nice to factor out devDependencies into the repo root to cut down on bootstrap time and storage requirements.  But for modules where we use the "bin" script that breaks things.  In this case I think adding it back into the package is the right move.

Eventually I'd like to have a `lerna bootstrap --hoist-external` or something that will let us define deps where they're actually used but then install them at the root and link bin scripts in where needed.  Some day. :)